docs/modules/serialization/pages/compact-serialization.adoc

```diff
@@ -60,13 +51,13 @@ extraction fails, Hazelcast throws an exception.
 Currently, Hazelcast supports extracting schemas out of classes that have the following
 field types:
 
-* Primitive types: `boolean`, `byte`, `short`, `int`, `long`, `float`, and `double`
-* Wrapper classes of primitive types: `Boolean`, `Byte`, `Short`, `Integer`, `Long`, `Float`, and `Double`
+* Primitive types: `boolean`, `byte`, `char`, `short`, `int`, `long`, `float`, and `double`
+* Wrapper classes of primitive types: `Boolean`, `Byte`, `Character`, `Short`, `Integer`, `Long`, `Float`, and `Double`
 * `String`
 * `java.time.LocalDate`, `java.time.LocalTime`, `java.time.LocalDateTime`, and `java.time.OffsetDateTime`
 * `java.math.BigDecimal`
 * Enums
-* Arrays of the types shown above
+* Array, `List`, `ArrayList`, `Set`, `HashSet`, `Map`, `HashMap` of the types shown above
```

Comment:
Maybe we can add a note about List is serialized/deserialized as ArrayList and so on. If users want another implementation they need to write custom serializer.

Reply:
Yes, I will do that, along with lots of other documentation updates with Compact before the GA. This PR is there to make sure the code snippets/docs we have are up to date